builtins/amp-ad.js

```diff
@@ -283,6 +283,10 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
+      if (!this.placeholder_) {
+        this.element.style.height = 0;
```

Comment:
1. Yes, I don't see why we shouldn't do `display:none`.
2. Malte is correct, all DOM mutations here should go via `this.deferMutate(callback)`. It ensures that (a) the changes are done in the mutate phase and (b) they don't interfere with scrolling.
3. `toggleFallback(true)` combined with `display = none` or `height = 0` makes no sense.